third-party/install_dependencies.sh

```diff
@@ -6,3 +6,4 @@ git submodule update --init --recursive
 ./install_emsdk.sh
 ./install_force-cover.sh
 ./install_mimalloc.sh
+./install_metis.sh
```

Comment:
can we make mimalloc install from not a subrepo (just clone a certain commit and then install?) also I think we can install benchmark using a debian package https://packages.debian.org/source/sid/benchmark